shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -276,6 +277,24 @@ public void removeFirstFrameListener(FirstFrameListener listener) {
         mFirstFrameListeners.remove(listener);
     }
 
+    /**
+     * Arranges this {@link SurfaceView} so that it's transparent and on top of its
+     * window.
+     */
+    public void enableOverlay() {
```

Comment:
I'm not sure that overlay is a great name for this.  First, Overlay has a meaning in Flutter which might confuse people.  But second, if the reason that someone is "enabling overlay" is because they want a transparent background, then I think the name confuses the intention.  So could we do something like

```
enableTransparentBackground()
disableTransparentBackground()
```

Or

```
setBackgroundColor(color) // and support a transparent color internally
```

Reply:
Done. Also clarified that the background color itself is still controlled by the Flutter UI.